lib/jobs/dell-wsman-simple-update-firmware.js

```diff
@@ -0,0 +1,249 @@
+                        logger.info('Retrieved component ids from packge.xml: ' + self.componentIds);
+                    }
+                });
+            })
+            .catch(function(err) {
+                logger.error('An error occurred while getting component from package.xml.', { error: err });
+                self.DeleteDownloadImage();
+                throw err;
+            });
+    };
+
+    /*
+     * Assemble request and call client request to update firmware
+     */
+    WsmanSimpleFirmwareUpdateJob.prototype._handleAsyncRequest = function() {
+        var self = this;
+        return Promise.try(function() {
+            logger.info('Assemble firmware update request on target: ' + self.targetConfig.serverAddress);
+            var request = {
+                serverAddress: self.targetConfig.serverAddress,
+                userName: self.targetConfig.username,
+                password: self.targetConfig.password,
+                fileName: self.imageFileName,
+                path: self.path
+            };
+            for(var i = 0; i < self.componentIds.length; i++) { //jshint ignore:line
+                request.componentId = self.componentIds[i];
+                logger.debug('Sending request to host: ' + self.apiServer + ' Dup API: ' +
+                    self.firmwareConfigs.dupUpdater + ' Request: ' + JSON.stringify(request));
+                self.clientRequest(self.apiServer, self.firmwareConfigs.dupUpdater, 'POST', request);
+            }
+            self.DeleteDownloadImage();
+        });
+    };
+
+    /*
+     * send request via wsman tool
+     */
+    WsmanSimpleFirmwareUpdateJob.prototype.clientRequest = function(host, path, method, data) {
+        var self = this;
+        var wsman = new WsmanTool(host, {
+            verifySSL: false,
+            recvTimeoutMs: 60000
+        });
+        return wsman.clientRequest(path, method, data, 'IP is invalid or httpStatusCode > 206')
+            .then(function(response) {
+                logger.debug('Repsonse body: ' + JSON.stringify(response.body));
+                if(response.body[0].status === 'undefined' || response.body[0].status === 'Failed' || response.body[0].status === '') {
+                    self._done(new Error("Failed to update firmware. Server: " + self.targetConfig.serverAddress));
+                } else {
+                    self._done();
+                }
+            })
+            .catch(function(error){
+                logger.error('An error occurred while processing simple firmware update.', { error: error });
+                self._done(error);
+            });
+    };
+
+    WsmanSimpleFirmwareUpdateJob.prototype.DeleteDownloadImage = function() {
+        logger.debug('Deleting downloaded image...');
+        var self = this;
+        try {
+            var childProcess;
+            if(fs.existsSync('/tmp/' + self.graphId)){
+                childProcess = new ChildProcess(
+                    'rm',
```

Comment:
Use node.js API `rmdir` rather  than hardcode the command:https://nodejs.org/api/fs.html#fs_fs_rmdir_path_callback